src/main/resources/interface-specification.yml

```diff
@@ -3952,6 +4271,98 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/producers/purposes':
```

Comment:
I would expect more something like 
`/purposes/eservice/{eserviceId}`
if it does not clash with other endpoints

Reply:
Seems good.